config/locales/en.yml

```diff
@@ -218,6 +218,83 @@ en:
       itself. (These scores are only available for English Wikipedia articles.)
     articles_short: Articles
 
+  items:
+    item_not_found: "This item has not been created yet."
```

Comment:
Are all these used, or is this just the fill list of messages that you found that currently use "article" terminology?

Using `item_not_found` (and so on) instead of sticking with `article_not_found` could introduce some additional complexity. I recommend staying with the same keys for each message except for using the `items` message tree instead of `articles`, so that it would either be `articles.article_not_found` or `items.article_not_found`.